src/test/groovy/com/google/gcloud/dsl/GCloudDslTest.groovy

```diff
@@ -0,0 +1,107 @@
+package com.google.gcloud.dsl
+
+import com.google.gradle.test.fixture.AppProjectBuilder
+import org.gradle.api.Project
+import org.junit.BeforeClass
+import org.junit.Test
+
+import static org.junit.Assert.*
+
+class GCloudDslTest {
+
+    static Project root
+    static Project child1
+    static Project child2
+
+    @BeforeClass
```

Comment:
I think this part is kind of expensive (project creation and setup).  All the tests are read only, so this is safe for now. I see what you're saying though.